frame/fast-unstake/src/mock.rs

```diff
@@ -164,12 +164,13 @@ impl Convert<sp_core::U256, Balance> for U256ToBalance {
 }
 
 parameter_types! {
-	pub static SlashPerEra: u32 = 100;
+	pub static DepositAmount: u32 = 28_000;
```

Comment:
why are we doing this? please keep the test accounts as simple as possible. Sounds like a irrelevant nit, but I have actually learned the hard way that it becomes a PITA very soon.